### DISCUSSION TITLE
Is it possible to deploy a single file without losing existent files?

### DISCUSSION THREAD

**Discussion by dmirica-gpsw** [JamesIves/github-pages-deploy-action#1486] [Q&A]
Context: gh pages is used to hold/serve test reports.
In time we gather a big number of reports and for deploying a new report we have to move around (make artifact and upload for deploy step) all of the current files and the new report. 
This starts to take too much time, so if it's possible to deploy a single file (in our case the new test report) without loosing the already existent/deployed files/reports would decrease the time needed.

**Comment by JamesIves** [comment]
Have you tried toggling the clean input to false?

**Comment by dmirica-gpsw** [comment]
Thanks for your answer, but meanwhile we moved to another solution entirely; i didn't had the chance to try the clean input, but the issue/question can be closed.

Thanks
